pkg/service/backup_go.go

```diff
@@ -33,6 +33,14 @@ func (b *BackupGo) BackupRun(
 	namespace string,
 	path string,
 ) (BackupHandler, error) {
+	ok, err := clusterHasRequiredNamespace(namespace, client.AerospikeClient())
```

Comment:
Won't it be better to validate the namespace either at startup or when modified using an API endpoint?